build-system-tests/scripts/mega-app-install.sh

```diff
@@ -75,25 +81,21 @@ while [[ $# -gt 0 ]]; do
     shift
 done
 
-# Check if MEGA_APP_NAME is provided
-if [[ -z "$MEGA_APP_NAME" ]]; then
```

Comment:
do we not need this check because we require it already on line 37, correct?

Reply:
Correct. the only time name is optional is in the initial setup step https://github.com/aws-amplify/amplify-ui/blob/e53c96d00db9d1f006d9f7050f707b16abcd6a5f/build-system-tests/scripts/setup-mega-app.sh#L82 
The name is either passed in or created in `setup-mega-app` and since the scripts are meant to be used together it doesn't make sense to check for it if it's required